specification.md

```diff
@@ -1055,10 +1057,168 @@ For example:
 }
 ```
 
-Note that the referenced function definition type in this case must be `rest` (default type).
+Note that the referenced function definition type in this case must be `openapi` (default type).
 
 For more information about functions, reference the [Functions definitions](#Function-Definition) section.
 
+#### Using functions for RESTful Service Invocations
+
+The specification also supports describing REST invocations in the [functions definition](#Function-Definition) using [OpenAPI Paths Object](https://spec.openapis.org/oas/v3.1.0#paths-object).
+
+Here is an example function definition for REST requests with method `GET` and request target corresponding with [URI Template](https://www.rfc-editor.org/rfc/rfc6570.html) `/users/{id}`:
+
+```json
+{
+  "functions":[
+    {
+      "name":"queryUserById",
+      "operation": {
+        "/users": {
+          "get": {
+            "parameters": [{
+              "name": "id",
+              "in": "path",
+              "required": true 
+            }]
+          }
+        }
+      },
+      "type":"rest"
+    }
+  ]
+}
+```
+
+Note that the [Function Definition](#Function-Definition)'s `operation` property must follow the OpenAPI Paths Object specification definition.
```

Comment:
link would be nice to have for OpenApi PAths object otherwise user has to google it themselves